test/jdk/javax/swing/JFileChooser/FileSystemView/ShellFolderStackOverflow.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please update copyright year. Same for the other file.

Reply:
Done!